modules/admin_manual/pages/installation/manual_installation/server_prep_ubuntu_20.04.adoc

```diff
@@ -23,6 +23,8 @@ in 2021. Consider upgrading to PHP 7.4. For more information see
 {php_versions_url}[Currently Supported PHP Versions]. This document considers two basic scenarios.
 You have/plan to upgrade from 18.04 LTS to 20.04 LTS or you have a fresh Ubuntu 20.04 LTS installation.
 
+IMPORTANT: PHP 8.0 is not supported by the ownCloud 10.x server series.
```

Comment:
To me, the words also imply (although they do not say directly) that PHP 8.0 will never be supported by the 10.x server series. I am not sure that the implication in my head is true. We might release `10.future` with PHP 8 support, or we might decide to number it 11.0 or? I just want to make sure that we do not mislead readers into thinking that PHP 8.0 support will never happen, or something.

Maybe a product owner/manager could comment about the words to use here? @pmaier1

Reply:
I did not mean the entire owncloud 10 server series, when I said 10.7 :-)
I agree with Phil, we should never say never, (php8 support may come faster than expected). I cannot tell if it will be a server 11.x or 10.x when php8 support gets added.

Reply:
We'll definitely not have it in the upcoming 10.8, correct? That's why I wasn't happy about only mentioning the lack of support for 10.7. How about a simple: PHP 8.0 is currently not supported by the ownCloud server.

Reply:
That sounds good - "currently" is the key word!

Reply:
@phil-davis Cool, but uhm, how do we fix it after my pre-mature commit here? Edit locally and make a new push?

Reply:
The easy way to adjust stuff is to do an online review, making a suggestion(s) and then GitHub gives you a chance to commit the suggestions. I did that just now.